modules/compliance-auto-update-remediations.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-advanced.adoc
+
+[id="automatically-update-remediations_{context}"]
+=  Automatically update remediations
+
+In some cases, a scan with newer content might mark remediations as `OUTDATED`. If the administrator wants to apply the remediations and remove the outdated ones, which will apply the newer ones, the `compliance.openshift.io/remove-outdated` can be applied.
+
```

Comment:
Is this a Procedure module? If so, you can add `.Procedure` with a single bullet (or two bullets because this isn't necessarily a sequence).